fsys.go

```diff
@@ -151,32 +154,31 @@ func (fs *fileServer) fsysproc() {
 	}
 }
 
-func fsysaddid(dir string, incl []string) *MntDir {
+// Add creates a new MntDir and returns a new reference to it.
```

Comment:
Acme used a linked list. But do we need to? There's one `MntDir` per executing external command right? So like 10? Seems like we could have a slice of `MntDir`? Conceivably that's just pointless code churn or maybe it makes the code simpler. What's your opinion?

Reply:
Yes, I agree a slice is good enough, but the code wouldn't be all that different. Instead, I think a `map[int]*MntDir` (where the key is the ID) would work really well here.

Reply:
Made the change to a map: https://github.com/rjkroege/edwood/pull/219